Competition/src/main/include/Muncher.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "ValorSubsystem.h"
+#include "Constants.h"
+
+#include <frc/XboxController.h>
+#include <frc/PWMVictorSPX.h>
+
+#ifndef MUNCHER_H
+#define MUNCHER_H
+
+class Muncher : public ValorSubsystem {
+    public:
+        Muncher();
+        void setController(frc::XboxController*);
+
+        void initMuncher();
+
+        void setDefaultState();
+        void assessInputs();
+        void assignOutputs();
+
+        void resetState();
```

Comment:
I don't see an implementation for this method?

Reply:
Azure failing due to this:
https://dev.azure.com/mray190/ViperbotsValor6800/_build/results?buildId=234&view=logs&j=34da5ad6-8f95-581f-335a-68aa6b6485c6&t=6f38a36f-aaa1-5232-1b05-bdfd007f39a1&l=134